gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/jupiter/reactor/processor/DefaultPlatformProcessorChainFactory.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.jupiter.reactor.processor;
+
+import io.gravitee.gateway.jupiter.core.processor.Processor;
+import io.gravitee.gateway.jupiter.reactor.processor.forward.XForwardForProcessor;
+import io.gravitee.gateway.jupiter.reactor.processor.tracing.TraceContextProcessor;
+import io.gravitee.gateway.jupiter.reactor.processor.transaction.TransactionProcessorFactory;
+import io.gravitee.gateway.report.ReporterService;
+import io.gravitee.node.api.Node;
+import io.gravitee.plugin.alert.AlertEventProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The default {@link AbstractPlatformProcessorChainFactory} used for synchronous HTTP requests
```

Comment:
This factory is also used for event native api, I would more say "used with default http dispatcher" or something like that.

Reply:
:+1: done